sql_user_check_redshift/database_stats.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -e
+
+port="5439"
+mode="stats"
+
+while getopts h:u:d:m:o:P:p flag
+do
+    case "${flag}" in
+        h) dbhost=${OPTARG};;
+        u) username=${OPTARG};;
+        d) database=${OPTARG};;
+        m) mode=${OPTARG};;
+        o) outputfile=${OPTARG};;
+        P) password=${OPTARG};;
+        p) port=${OPTARG};;
+    esac
+done
+
+export PGPASSWORD=$password
+
+echo "dbhost: $dbhost username: $username database: $database port: $port mode: $mode outputfile: $outputfile";
+
+# Check if psql command is available
+if ! command -v psql &>/dev/null; then
+    echo "psql is installed and available."
+    exit 1
+fi
+
+if [ -z "$dbhost" ] || [ -z "$username" ] || [ -z "$database" ] || [ -z "$outputfile" ]; then
```

Comment:
Database name should optional here since user may not even know which databases they are going to connect.

Reply:
>>> Connect to postgres database
for this we need to specify a database in psql command, so it should be a mandatory input argument for the script ??

Reply:
there is no postgres database in redshift.
default database is dev, but got this error when trying to execute queries on it.

psql: error: connection to server at "redshift-cluster-2.czrxemvpvasi.us-east-2.redshift.amazonaws.com" (3.129.197.168), port 5439 failed: FATAL:  Cannot connect to shared database "awsdatacatalog". Connect to the databases in your cluster instead and use cross-database query notation <shareddatabase>.<schema>.<object> to query the data in shared database.